skeleton/addons/aws/modules/iam_groups/main.tf

```diff
@@ -18,6 +18,7 @@ resource "aws_iam_group_policy_attachment" "admin_access" {
   policy_arn = data.aws_iam_policy.admin_access.arn
 }
 
+# tfsec:ignore:aws-iam-no-policy-wildcards
```

Comment:
```suggestion
# Policy from https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_examples_aws_my-sec-creds-self-manage.html
# tfsec:ignore:aws-iam-no-policy-wildcards
```

Should we add the source before the wild card? (for curious people like me 😇 )

Thanks

Reply:
Thanks, added in https://github.com/nimblehq/infrastructure-templates/pull/210/commits/64aa951af28711669058a95ccdb35f84e6a1aa1a